### DISCUSSION TITLE
[Paper Suggestion]: RLP: Reinforcement as a Pretraining Objective

### DISCUSSION THREAD

**Discussion by NicklasOsterbacka** [dsai-nlp/core#25] [Paper Suggestions]
### Paper Title

RLP: Reinforcement as a Pretraining Objective

### Paper Link

https://arxiv.org/abs/2510.01265

### Brief Summary

The paper introduces reinforcement learning as a pre-training objective. The core is that they treat chain-of-thought as an exploratory action, with the model being rewarded based on the information gain provided for predicting future tokens. Their formulation yields a verifier-free dense reward signal, and their benchmark results are improved compared to baselines.

### Relevance to Group

This paper outlines a way to integrate CoT-based reasoning much earlier in the training process than traditional fine-tuning through SFT or RL.

### Voting Instructions (do not edit)

```markdown
---
## 🗳️ Voting Instructions
To vote for this paper, **click the ↑ upvote button in the top-left corner of this discussion post**
We'll prioritize papers with the most votes!  

↓ Right here
```
